docs/import-data.rst

```diff
@@ -4,31 +4,44 @@
 Import data
 ===========
 
-This guide demonstrates how to import data into your CrateDB Cloud
-cluster. This how-to assumes that you already have a cluster running. If you
-need help starting a cluster, please see :ref:`tutorial on how to deploy a
-cluster for the first time <cloud-tutorials:cluster-deployment>`.
+This guide shows two methods of importing data into your CrateDB Cloud
+cluster. One is available directly in your Cloud Console, and the other in the
+Admin UI of your cluster.
 
 .. rubric:: Table of contents
 
 .. contents::
    :local:
 
+Import data from URL
+--------------------
 
-.. _AVC-d:
+The more convenient method for importing data is to import from a URL
+directly in your Cloud Console. See details `here`_.
+
+Import data in the Admin UI
+---------------------------
+
+If your data isn't publicly available or is in a format unsupported by URL
+import in the Cloud Console, you will have to import manually from the Admin 
+UI.
+
+.. _prerequisite:
 
 Pre-requisite
--------------
+'''''''''''''
 
-To import data into your cluster, you need to have access to your cluster's
-Admin UI. The username is always ``admin`` and the password is set upon cluster
-deployment.
+To import data this way, you need to have access to your cluster's
+Admin UI. The username is always ``admin`` and the password is set upon 
+cluster deployment. Make sure to store it securely.
+
+.. _create-table:
 
 Create a table
---------------
+''''''''''''''
 
-Before you import the data, the table that will be filled with the data must be
-created. Once logged in to the Admin panel, navigate to the console.
+Before you import the data, the table that will be filled with the data must 
```

Comment:
To the best of my knowledge, I'm don't think this is true. The table will be created on-the-fly even if no table exists.